mm2src/mm2_main/src/lp_swap/recreate_swap_data.rs

```diff
@@ -528,7 +529,7 @@ mod tests {
2f394729d0ce838f14e1e76117c001210315d9c51c657ab1be4ae9d3ab6e76a619d3bccfe830d5363fa168424c0d044732ffffffff03a2296b050000000017a91491c45f69e1760c12a1f90fb2a811f6dfde35cc35870000000000000000166a144da9e7080175e8e10842e0e161b33cd298cab30bac503d64000000001976a9141462c3dd3f936d595c9af55978003b27c250441f88ac09ebb061000000000000000000000000000000","tx_hash":"6287e0d30951cd859bfb837eb1e5409f7596e75ffeb2e61fd6df1843bfd0203d"}}},{"timestamp":1638984457230,"event":{"type":"MakerPaymentWaitConfirmStarted"}},{"timestamp":1638984504262,"event":{"type":"MakerPaymentValidatedAndConfirmed"}},{"timestamp":1638984504262,"event":{"type":"TakerPaymentSent","data":{"tx_hex":"0400008085202f8901bcb570da43050193f6dca5c5260be8f7668887943b644341018e9ec76791b4fc010000006b483045022100fe6c90568a256b531bcd18321c15b3ce68c2d5d603768dea6aba68dcc170b801022076a34c006a92786bcdee6a1dcb46947fb49911e4f51ec27e880c3396d64d59b2012103b1e544ce2d860219bc91314b5483421a553a7b33044659eff0be9214ed58adddffffffff0300e1f5050000000017a9145a6125d597d2ce37bde9983d4d9d481335139bc4870000000000000000166a144da9e7080175e8e10842e0e161b33cd298cab30b8cdef305000000001976a91483762a373935ca241d557dfce89171d582b486de88ac37ebb061000000000000000000000000000000","tx_hash":"667dea45dbd36e1f15ee8170e73e2afefd0a15d47d3bcd65655ca71d838445a2"}}},{"timestamp":1638984594337,"event":{"type":"TakerPaymentSpent","data":{"transaction":{"tx_hex":"0400008085202f8901a24584831da75c6565cd3b7dd4150afdfe2a3ee77081ee151f6ed3db45ea7d6600000000d74730440220422edb8ef5cd3991eb309c3a4fa5fe5d9ffe08d3d6b4b789c5587061d7993864022049cd082398f5a37a9e7411d56976e61bcce9162d0f5f1fb24e40bcf2f4ec0052012023a6bb64bc0ab2cc14cb84277d8d25134b814e5f999c66e578c9bba3c5e2d3a4004c6b63047009b161b1752103b1e544ce2d860219bc91314b5483421a553a7b33044659eff0be9214ed58adddac6782012088a9144da9e7080175e8e10842e0e161b33cd298cab30b88210315d9c51c657ab1be4ae9d3ab6e76a619d3bccfe830d5363fa168424c0d044732ac68ffffffff0118ddf505000000001976a9141462c3dd3f936d595c9af55978003b27c250441f88ac7009b161000000000000000000000000000000","tx_hash":"ab1eb5b65a302370af2607e0b64b60fc04360de33a87799bca1dcf337344b616"},"secret":"23a6bb64bc0ab2cc14cb84277d8d25134b814e5f999c66e578c9bba3c5e2d3a4"}}}],"maker_amount":"0.9090909090909090909090909090909090909090909090909090909090909090909090909090909090909090909090909091","maker_coin":"RICK","taker_amount":"1","taker_coin":"MORTY","gui":null,"mm_version":"","success_events":["Started","Negotiated","TakerFeeSent","MakerPaymentReceived","MakerPaymentWaitConfirmStarted","MakerPaymentValidatedAndConfirmed","TakerPaymentSent","WatcherMessageSent","TakerPaymentSpent","MakerPaymentSpent","Finished"],"error_events":["StartFailed","NegotiateFailed","TakerFeeSendFailed","MakerPaymentValidateFailed","MakerPaymentWaitConfirmFailed","TakerPaymentTransactionFailed","TakerPaymentWaitConfirmFailed","TakerPaymentDataSendFailed","TakerPaymentWaitForSpendFailed","MakerPaymentSpendFailed","TakerPaymentWaitRefundStarted","TakerPaymentRefunded","TakerPaymentRefundFailed"]}"#;
```

Comment:
Could you please move these big JSON payloads to files similar to https://github.com/KomodoPlatform/atomicDEX-API/blob/5e2dd3dd716d19fb99441d5a3d7ec185a120153a/mm2src/coins/tx_history_storage/tx_history_v2_tests.rs#L10 ?